docs/Examples.md

```diff
@@ -445,6 +452,33 @@ spec:
         - "/mnt/apps/app1.tgz"
         - "/mnt/apps/app2.tgz"
 ```
+In the above example, app1 and app2 are installed on Splunk Standalone instances.
```

Comment:
Move this to 436:

In the standalone example, app1 and app2 are installed on Splunk Standalone instances.